.gitignore

```diff
@@ -102,8 +102,12 @@ certbot.log
 rerun.txt
 
 # Ignore VCR
+<<<<<<< HEAD
```

Comment:
These lines with "<<<" and "===" mean that there was a merge conflict in your changes.
I don't think this file needs to be modified for this PR, so can you restore it?